modules/javafx.controls/src/main/java/javafx/scene/control/SkinBase.java

```diff
@@ -219,6 +237,7 @@ protected final void consumeMouseEvents(boolean value) {
      *  may be {@code null}
      * @since 9
      */
+    // TODO I would like to deprecate and remove these methods, and replace them by listenerHelper().add**()
     protected final void registerChangeListener(ObservableValue<?> observable, Consumer<ObservableValue<?>> operation) {
```

Comment:
I don't think we want to deprecate this. It is part of the public API, so there would need to be a compelling reason to deprecate it.

Reply:
we can keep it, eventually replacing the implementation with ListenerHelper, see [JDK-8296076](https://bugs.openjdk.org/browse/JDK-8296076)